user.js

```diff
@@ -675,49 +675,55 @@ module.exports = function user (options) {
 
       login.save$(function (err, login) {
         if (err) return done(err)
-        seneca.act({role: role, cmd: 'set_user_lock', id: user.id}, function (err, out) {
+        cmd_increment_lock(seneca, user.id, true, function (err, out) {
           if (err) return done(err)
+
           seneca.log.debug('login/ok', why, user, login)
           done(null, {ok: true, user: user, login: login, why: why})
```

Comment:
i'll make a separate task related to cleaning returned data as it needs more detailed full code check. That code area is not related to changes made by the brute force implementation